src/helper/client.cpp

```diff
@@ -170,6 +183,21 @@ bool client::handle_command(network::request_init::request &command)
         return false;
     }
 
+    //    if (!compute_extension_status()) {
+    //        auto response_cf =
+    //            std::make_shared<network::config_features::response>();
+    //        response_cf->enabled = false;
+    //
+    //        SPDLOG_DEBUG("sending config_features to request_init");
+    //        try {
+    //            return broker_->send(response_cf);
+    //        } catch (std::exception &e) {
+    //            SPDLOG_ERROR(e.what());
+    //        }
+    //
+    //        return true;
+    //    }
+
```

Comment:
This logic will be used as part of the next PR.  It was commented out in order to make this PR smaller and get it through faster